### PR TITLE
Change floor level to being in native space

### PR DESCRIPTION
**Pull request by Manishearth**
It already is, but it's more clearer this way. I got confused with this twice :p

### CODE REVIEW

**Comment by Manishearth** [comment]
I think that's unnecessary indirection; the XR API talks about everything in native space, we should too.

> is that it’s a transform from the local/viewer space (I.e the headset is 0,0,0) to the floor space (I.e the floor is 0,0,0)

THis value changes over time, that's the source of the confusion. You're talking about the viewerOrigin at t=0.01 (the thing set from FakeXRDeviceInit), which is not a concept we otherwise use.

**Comment by alcooper91** [comment]
Okay, I think the key thing that I was missing here is that the webxr spec has some specific notions of origin that were a bit confusing to me.  I think if we update the comment above FakeXRRigidTransformInit to something like:
When used as an object's origin, it represents the transform from that object's origin space to the native space

We may be able to avoid some further confusion in the future.

File: explainer.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ dictionary FakeXRDeviceInit {
     boolean supportsUnbounded = false;
     // The bounds coordinates. If null/empty, bounded reference spaces are not supported. If not, must have at least three elements.
     sequence<FakeXRBoundsPoint> boundsCoodinates;
-    // A transform used to identify the physical position of the user's floor relative to the viewerOrigin at initialization.  If not set, indicates that the device cannot identify the physical floor.
-    FakeXRRigidTransformInit localToFloorLevelTransform;
+    // A transform used to identify the physical position of the user's floor.  If not set, indicates that the device cannot identify the physical floor.
+    FakeXRRigidTransformInit floorOrigin;
```
Comment (alcooper91) [inline explainer.md]: Can you also update the set/clear LocalToFloorLevelTransform methods?

**Pull request merged**